InfluxDb/app.json

```
@@ -0,0 +1,10 @@
+{
+    "appid": "5e1e914cba38acaa83d4e39139dadbec478fbb6e",
+    "name": "InfluxDb",
```

The name here is also wrong. It is correct in the app description.
```suggestion
    "name": "InfluxDB",
```